model/jpa/src/main/java/org/keycloak/models/jpa/entities/UserEntity.java

```diff
@@ -96,17 +97,17 @@
     @Column(name = "REALM_ID")
     protected String realmId;
 
-    @OneToMany(cascade = CascadeType.REMOVE, orphanRemoval = true, mappedBy="user")
+    @OneToMany(cascade = CascadeType.REMOVE, orphanRemoval = true, mappedBy="user", fetch = FetchType.LAZY)
```

Comment:
Please remove this and the two same changes below - LAZY is the default fetch strategy anyway.